final release/model_sklearn.py

```diff
@@ -0,0 +1,14 @@
+import keras 
```

Comment:
Are you only going to have `sklearn` in this file or will you store all the models in it? I question `model_sklearn.model_sklearn` and instead suggest `models.sklearn`. Or even more specific like `models.svm`.

Reply:
Just Sklearn. Could I put everything in there if you want? How about model_sklearn.svm?

Reply:
I think it makes sense to have everything in there.